parts/k8s/kuberneteswindowssetup.ps1

```diff
@@ -255,6 +255,11 @@ try
         Initialize-DataDirectories
 
         New-Item -ItemType Directory -Path "c:\k" -Force | Out-Null
+        icacls.exe "c:\k" /inheritance:r

```

Comment:
A log should be helpful to check whether the folder has this fix.

Reply:
I added some logs.

This was in c:\AzureData\CustomDataSetupScript.log:
![image](https://user-images.githubusercontent.com/18291632/131537707-1ac44ea5-e67e-407a-93e4-36ad2bfeb6fb.png)